File: HLC01_03Java/src/main/java/org/example/reservaHotel02/Console.java
package org.example.reservaHotel02;

import java.util.Scanner;

public class Console {
    private final Scanner scanner = new Scanner(System.in);

    public void showMessage(String message, boolean jumpLine) {
        if (jumpLine) {
            System.out.println(message);
        } else {
            System.out.print(message);
        }
    }

    public String readString() {
        return scanner.nextLine();
    }

}

File: HLC01_03Kotlin/src/main/kotlin/reservaHotel02/Console.kt
package org.example.reservaHotel02


class Console {

    fun showMessage(message: String, jumpLine: Boolean = true) {
        if (jumpLine) {
            println(message)
        } else {
            print(message)
        }
    }


    fun readString(): String {
        return readln()
    }

}